x-pack/plugins/serverless_search/kibana.jsonc

```diff
@@ -32,10 +32,11 @@
       "searchHomepage",
       "searchIndices",
       "searchInferenceEndpoints",
-      "usageCollection"
+      "usageCollection",
+      "searchApiKeys"
```

Comment:
is it needed here or only needed in search_indices plugin? AFAIC, its only used in search_indices

Reply:
Guess you added this accidentally or "kbn bootrsrap" did it